src/pages/Home/VideoItem.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Card, Grid, Image } from 'semantic-ui-react';
+
+function VideoItem(props) {
+    const {video} = props;
+    return (
+        <Grid.Column>
+            <Card href='#' className="selectable">
+                <Image alt={'thumbnail'} src={video?.snippet?.thumbnails?.high?.url} />
+                <Card.Content>
+                    <Card.Header>{video?.snippet?.title}</Card.Header>
+                    <Card.Meta>{video?.snippet?.channelTitle}</Card.Meta>
+                    <Card.Description>
+                        {video?.snippet.description}
+                    </Card.Description>
+                </Card.Content>
+            </Card>
```

Comment:
This `Card` needs several properties that are obtained from `video.snippet`, instead of having `video.snippet` several times in the file, by destructuring the object this can change to:

```jsx
 const { title, channelTitle, description, thumbnails } = video?.snippet ?? {};

...
            <Card href='#' className="selectable">
                 <Image alt={'thumbnail'} src={thumbnails.high?.url} />
                 <Card.Content>
                     <Card.Header>{title}</Card.Header>
                     <Card.Meta>{channelTitle}</Card.Meta>
                     <Card.Description>
                         {description}
                     </Card.Description>
                 </Card.Content>
             </Card>
...
```